src/Sulu/Bundle/AdminBundle/Resources/js/components/Form/grid.scss

```diff
@@ -5,7 +5,7 @@
 
 .grid {
     margin-bottom: -$sectionMarginBottom;
-    overflow-y: hidden;
+    contain: layout;
```

Comment:
Is not supported by safari but actually safari does not have the bug why it was introduced and is handling the `margin-bottom: -` correctly.